Specs/Scene/BatchTableHierarchySpec.js

```diff
@@ -287,6 +387,25 @@ describe("Scene/BatchTableHierarchy", function () {
     expect(hierarchy.getProperty(2, "coordinates")).toEqual(position);
   });
 
+  it("setProperty works with binary values", function () {
```

Comment:
deuplicate test with the one above it?

Reply:
No, the constructor passes in input with binary IDs. I'll change the name of the test to describe this better.